plugins/couchbase/couchbaseembed.cpp

```diff
@@ -364,7 +383,7 @@ namespace couchbaseembed
         const char *user = "";
         const char *password = "";
         const char *bucketname = "default";
-        unsigned port = 8093;
+        unsigned port = 8091;
```

Comment:
Because that's the default couchbase port, the point of this change is to use the port provided in config if one exists. Doesn't make sense to denote in our portlist.h, this is a couchbase port